src/checkstatus/get_dbstatus_function.py

```diff
@@ -2,9 +2,14 @@
 import boto3
 import constants
 import utility as util
+import logging
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
 
 def lambda_get_dbinstance_status(event, context):
     """Method to obtain status of a RDS instance post actions such as snapshot creation, rename, restore or delete"""
+    logger.info('## starting function execution ...')
```

Comment:
this and the "ending execution" logs can be removed for all top level handler methods.
Retain them for any internal methods used
(to be followed across all files)